src/Cake.Common/Tools/OpenCover/OpenCoverAliases.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using Cake.Core;
+using Cake.Core.Annotations;
+
+namespace Cake.Common.Tools.OpenCover
+{
+    /// <summary>
+    /// Contains functionality related to OpenCover.
+    /// </summary>
```

Comment:
I think we need a `see` tag here.  I added what I thought would be the right thing here:

https://github.com/cake-build/cake/blob/develop/src/Cake.Common/Tools/GitVersion/GitVersionAliases.cs#L9

But looking at the ReportGenerator PR here:

https://github.com/cake-build/cake/pull/467/files#diff-cfdf56b7f17f3d21961b1f1fa250f20dR10

I maybe didn't do it in the right way.  I think we need to some sort of consensus here on the `default` href's that we want to include for each Addin.